README.md

```diff
@@ -63,6 +53,10 @@ To configure the SalemoveSDK, follow the steps:
 3. [Configuring the project](#project-configuration)
 4. [Configuring the sdk](#sdk-configuration)
 
+### Push Notifications
+
+In order for the project to receive chat messages and other events, [push capabilities][3] should be enabled for the target. SaleMove will be using your certificate as a gateway to send the messages to the hosting application.
```

Comment:
SaleMove will use

Reply:
SaleMove will use your certificate when sending messages via Apple Push Notification Service to the SaleMove client in your app.
(I think that is correct - please check)